cartography/intel/github/util.py

```diff
@@ -84,31 +84,27 @@ def fetch_all(
     while has_next_page:
         try:
             resp = fetch_page(token, api_url, organization, query, cursor)
+            retry = 0
         except requests.exceptions.Timeout:
             retry += 1
-            if retry >= retries:
-                logger.error(
-                    f"GitHub: Could not retrieve page of resource {resource_type} due to API timeout.",
-                    exc_info=True,
-                )
-                raise
-            else:
-                time.sleep(1 * retry)
-                continue
         except requests.exceptions.HTTPError:
             retry += 1
+        except requests.exceptions.ChunkedEncodingError:
+            retry += 1
+        finally:
             if retry >= retries:
                 logger.error(
                     f"GitHub: Could not retrieve page of resource `{resource_type}` due to HTTP error.",
                     exc_info=True,
                 )
                 raise
-            else:
+            elif retry > 0:
                 time.sleep(1 * retry)
-                continue
```

Comment:
I think we still need this continue to actually retry the request?

Reply:
thanks for catching this